src/utils/browser.js

```diff
@@ -49,3 +51,19 @@ export const getCookie = name => {
   }
   return decodeURI(dc.substring(begin + prefix.length, end));
 };
+
+export const buildDynamicURL = (url, qsParams, dynamicQsValues = []) => {
+  const keys = Object.keys(qsParams);
+  keys.length && (url += '?');
+  keys.forEach(key => {
+    const param = qsParams[key];
+    //eslint-disable-next-line
+    if (!isEvaluateParam(param) || dynamicQsValues[param.replace(/[\{\}]/g, '')]) {
```

Comment:
Redundant character escape '\{' in RegExp --> `dynamicQsValues[param.replace(/[{}]/g, '')])`
And then I guess you can remove the `//eslint-disable-next-line`